deployments/atd_service_bot_socrata.yaml

```diff
@@ -0,0 +1,62 @@
+###
+### A complete description of a Prefect Deployment for flow 'Service Bot: Issues to Socrata'
+###
+name: 'Service Bot: Issues to Socrata'
+description: |-
+  Repo: https://github.com/cityofaustin/atd-service-bot, Uploads (replaces) github issue data from our atd-data-tech repo
+  to an Open Data Portal dataset (AKA Socrata)
+version: 8adc901677047f8853f3ba8e94fa8880
+# The work queue that will handle this deployment's runs
+work_queue_name: atd-service-bot
+work_pool_name: atd-data-03
+tags: []
+parameters: {}
+schedule:
+  cron: 21 5 * * *
+  timezone: null
+  day_or: true
+is_schedule_active: false
+infra_overrides: {}
+infrastructure:
+  type: process
+  env: {}
+  labels: {}
+  name: null
+  command: null
+  stream_output: true
+  working_dir: null
+  block_type_slug: process
+  _block_type_slug: process
+
+###
+### DO NOT EDIT BELOW THIS LINE
+###
+flow_name: 'Service Bot: Issues to Socrata'
+manifest_path: null
+storage:
+  repository: https://github.com/cityofaustin/atd-prefect
+  reference: md-11418-service-bot-socrata
+  access_token: '**********'
```

Comment:
I found the `atd-service-bot-staging` block which is where this access token appears to be stored in the prefect web UI. I find it interesting that it's represented in this file with a masked value instead of a boolean indicating that it simply exists, so that's what got me thinking about it. 

I don't see any problem with prefect having a value it can presumably pass down to a running flow to use to get files from GitHub over an auth'd connection, but I guess it seems different to me since the other flows that use a GH backed storage block don't have one. 

Does this need to be here because the script which is called by this flow need it to publish issues as `atdservicebot`? 

I'm sorry if this question is uniformed. I'm sure this is fine, and I am curious more than anything.

Reply:
Not at all! Just saw this but it relates to [this thread](https://austininnovation.slack.com/archives/CHZE6BC6L/p1670439656261679) and an issue that Charlie ran into in Prefect v1 too. In that case, this part happened in the code.